index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>MATKA '17</title>
+    <meta name="description" content="Quark is proud to present the biggest college gaming event in India - MATKA!" />
+    <meta name="keywords" content="quark, 2017, 17, bits, pilani,bits-quark, annual, techfest, technical, festival, bpgc, matka" />
+    <meta name="author" content="Quark 2016" />
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Black+Ops+One" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Russo+One" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Bungee" rel="stylesheet">
+    <!--     <link href="https://fonts.googleapis.com/css?family=Scope+One" rel="stylesheet"> -->
+    <!-- /Fonts -->
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="public/stylesheets/bootstrap.min.css">
+    <link rel="stylesheet" href="public/stylesheets/style.css">
+    <!-- /Stylesheets -->
+</head>
+
+<body>
+    <!-- Navigation Bar -->
+    <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+        <div class="container">
+            <a class="navbar-brand" href="#">MATKA</a>
+            <ul class="nav navbar-nav">
+                <li>
+                    <a href="#">Q'17</a>
+                </li>
+                <li>
+                    <a href="#">Sponsors</a>
+                </li>
+                <!--                 <li>
+                    <a href="#">Contacts</a>
+                </li> -->
+                <li>
+                    <a href="#">Register</a>

```

Comment:
Add the links, before I can accept this pull request. This is because MATKA is directly linked to this repo.